src/HoneybadgerClient.php

```diff
@@ -43,12 +41,16 @@ public function notification(array $notification): array
                 'notices',
                 ['body' => json_encode($notification, JSON_PARTIAL_OUTPUT_ON_ERROR)]
             );
-        } catch (Exception $e) {
-            throw ServiceException::generic();
+        } catch (Throwable $e) {
```

Comment:
Might just want to double check what our minimum PHP support level is and ensure `Throwable` is available. When this was built we supported PHP before `Throwable`. Just something to look out for, I think we're fine though!

Reply:
Yeah, did that. It's PHP 7